src/components/CallView/shared/LocalAudioControlButton.vue

```diff
@@ -148,6 +154,26 @@ export default {
 				this.model.disableAudio()
 			} else {
 				this.model.enableAudio()
+
+				// Lowers hand when mic is on for 3 seconds
+				if (this.lowerHandTimeout !== null) {
+					clearTimeout(this.lowerHandTimeout)
+					this.lowerHandTimeout = null
+				}
+				if (this.model.attributes.raisedHand?.state) {
+					this.lowerHandTimeout = setTimeout(() => {
+						if (this.showMicrophoneOn) {
+							this.model.toggleHandRaised(false)
+							this.$store.dispatch(
+								'setParticipantHandRaised',
+								{
+									sessionId: this.$store.getters.getSessionId(),
+									raisedHand: this.model.attributes.raisedHand,
+								}
+							)
+						}
```

Comment:
Lowering/raising hand has a particular process for breakout rooms. I suggest to use the method in `TopBarMenu`
https://github.com/nextcloud/spreed/blob/main/src/components/TopBar/TopBarMenu.vue#L453-L478